Bash_scripts/HW17/Dockerfile_hw17

```diff
@@ -0,0 +1,72 @@
+FROM python:3.7-slim-buster
+
+LABEL DEBIAN_FRONTEND="noninteractive"
+LABEL TERM="linux"
+
+ENV AIRFLOW_VERSION=1.10.9 
+ENV AIRFLOW_USER_HOME=/usr/local/airflow 
+
+LABEL AIRFLOW_DEPS="" 
+LABEL PYTHON_DEPS="" 
+
+ENV AIRFLOW_HOME=${AIRFLOW_USER_HOME}
+ENV LANGUAGE="en_US.UTF-8" 
+ARG	LANG="en_US.UTF-8"
+ARG LC_ALL="en_US.UTF-8" 
+ENV LC_CTYPE="en_US.UTF-8" 
+ENV LC_MESSAGES="en_US.UTF-8" 
+
+ARG buildDeps=' \
+        freetds-dev \
+        libkrb5-dev \
+        libsasl2-dev \
+        libssl-dev \
+        libffi-dev \
+        libpq-dev \
+        git'; \
+		set -ex 
```

Comment:
Is it for ARG?

Reply:
it is RUN, build completed successfully.
https://i.imgur.com/t98GKWd.png